xonsh/xoreutils/_which.py

```diff
@@ -295,7 +287,10 @@ def whichall(command, path=None, verbose=0, exts=None):
         not a VisualBasic script but ".vbs" is on PATHEXT. This option
         is only supported on Windows.
     """
-    return list( whichgen(command, path, verbose, exts) )
+    if verbose:
+        return list( whichgen(command, path, verbose, exts) )
+    else:
+        return list( absName for absName, _ in whichgen(command, path, verbose, exts))
```

Comment:
True, but am just following the existing naming convention of this old module. The same variables are used other places with these names